src/controllers/arraysControllers.js

```diff
@@ -0,0 +1,49 @@
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./../lib/arrays');
+
+exports.elementAtID = (req, res) => {
```

Comment:
Additional info - you can deconstruct off parameters too eg.

exports.elementAtID = ({ body, params }, res) => {

the whole object is the req and if you know it is going to have the keys body and params you can destruct straight into your function.